server/models/user.js

```diff
@@ -28,7 +28,7 @@ userSchema.pre('save', function(next){
     })
 })
 
-userSchema.statics.comparedPassword = function(candidatePassword, cb) {
+userSchema.methods.comparedPassword = function(candidatePassword, cb) {
```

Comment:
can you change it back to statics, so I can prove this PR!(I am really willing to) thank you buddy!